Solutions/SonicWall Firewall/Hunting Queries/OutboundSSHConnections.yaml

```diff
@@ -0,0 +1,37 @@
+id: dedb8fb9-3caa-4b00-ae88-1898eed78917
+name: Outbound SSH/SCP Connections
+description: |
+  'This query looks for outbound SSH/SCP connections identified by the expected port number (22) or by the SonicWall Deep Packet Inspection services.

```

Comment:
Hi @jaimeesc, 
Everything is good. I have tested the solution. Just we need this Hunting Query Description below 255 characters.